ticdc/ticdc-open-api.md

```diff
@@ -0,0 +1,617 @@
+                65,
+            ],
+            "table_operations": {}
+        }
+    ]
+}
+```
+
+## Pause a replication task
+
+This is an asynchronous interface. If the request is successful, `202 Accepted` is returned. The returned result only means that the server agrees to run the command but does not guarantee that the command will be run successfully.
+
+### Request URI
+
+`POST /api/v1/changefeeds/{changefeed_id}/pause`
+
+### Parameter description
+
+#### Path parameters
+
+| Parameter name | Description |
+| :-------------- | :----------------------------------- |
+| `changefeed_id` | The ID of the replication task (changefeed) to be paused. |
+
+### Example
+
+The following request pauses the replication task with the ID `test1`.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST http://127.0.0.1:8300/api/v1/changefeeds/test1/pause
+```
+
+If the request is successful, `202 Accepted` is returned. If the request fails, an error message and error code are returned.
+
+## Resume a replication task
+
+This is an asynchronous interface. If the request is successful, `202 Accepted` is returned. The returned result only means that the server agrees to run the command but does not guarantee that the command will be run successfully.
+
+### Request URI
+
+`POST /api/v1/changefeeds/{changefeed_id}/resume`
+
+### Parameter description
+
+#### Path parameters
+
+| Parameter name | Description |
+| :-------------- | :----------------------------------- |
+| `changefeed_id` | The ID of the replication task (changefeed) to be resumed. |
+
+### Example
+
+The following request resumes the replication task with the ID `test1`.
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X POST http://127.0.0.1:8300/api/v1/changefeeds/test1/resume
+```
+
+If the request is successful, `202 Accepted` is returned. If the request fails, an error message and error code are returned.
+
+## Query the replication subtask list
+
+This is a synchronous interface. If the request is successful, the basic information of all replication subtasks is returned.
+
+### Request URI
+
+`GET /api/v1/processors`
+
+### Example
+
+{{< copyable "shell-regular" >}}
+
+```shell
+curl -X GET http://127.0.0.1:8300/api/v1/processors
+```
+
+```json
+[
+    {
+        "changefeed_id": "test1",
+        "capture_id": "561c3784-77f0-4863-ad52-65a3436db6af"
+    }
+]
+```
+
+## Query a specific replication subtask
+
+This is a synchronous interface. If the request is successful, the detailed information of the specified replication subtask (`processor`) is returned.
+
+### Request URI
+
+`GET /api/v1/processors/{changefeed_id}/{capture_id}`
+
+### Parameter description
+
+#### Path parameters
+
+| Parameter name | Description |
+| :-------------- | :----------------------------------- |
+| `changefeed_id` | The changefeed ID of the replication subtask to be queried. |
+| `capture_id` | The capture ID of the replication subtask to be queried. |
+
+### Example
+
```

Comment:
May be it is better that we add some describe about this example:

The following request query the detail information of a subtask which `changefeed_id` is `test` and `capture_id` is `561c3784-77f0-4863-ad52-65a3436db6af`. (A subtask can be indentifed by `changefeed_id` and `capture_id`).

Reply:
```suggestion
The following request queries the detailed information of a subtask whose `changefeed_id` is `test` and `capture_id` is `561c3784-77f0-4863-ad52-65a3436db6af`. (A subtask can be indentifed by `changefeed_id` and `capture_id`).
```